openapi/parameters/common-crs/bbox-crs.yaml

```diff
@@ -1,14 +1,20 @@
-name: {bbox-parameter-name}-crs
+name: inputCrs
 description: |-
-  Asserts the CRS used for the coordinate values of the bbox parameter. 
+  Asserts the CRS used for the coordinate values of a bounding
+  box input parameter. 
 in: query
 required: false
 schema:
-  type: string
-  format: uri
-  enum:
-    - http://www.opengis.net/def/crs/OGC/1.3/CRS84
-  default:
-    - http://www.opengis.net/def/crs/OGC/1.3/CRS84
-style: form
-explode: false
+  type: object
+  required:
+    - {bbox-input-name}
+  properties:
+    {bbox-input-name}:
```

Comment:
Shouldn't this be a `additionalProperties`, or `patternProperties` with `.+[crs]`?
This definition would look for an explicit `{bbox-input-name}` property.

I think the solution would be: 

```yaml
in: query
name: inputs  # just a generic name to avoid ambiguity, ignored since 'explode: true'
explode: true
style: form
schema: 
  type: object
  patternProperties:
    "^.+[crs]$":
      type: string  
      format: uri  # etc. other params...
```

Ideally, we would have a separate schema for the following content, such that we can simply `$ref` to it under the pattern. https://github.com/opengeospatial/ogcapi-processes/blob/d011fe92750a0843106fcb87b6292db6e2b0fa51/openapi/schemas/processes-core/bbox.yaml#L16-L22

Reply:
Done.